tests/hotplug/cpu.go

```diff
@@ -52,7 +48,13 @@ var _ = Describe("[sig-compute][Serial]CPU Hotplug", decorators.SigCompute, deco
 			WorkloadUpdateMethods: []v1.WorkloadUpdateMethod{v1.WorkloadUpdateMethodLiveMigrate},
 		}
 		patchWorkloadUpdateMethod(originalKv.Name, virtClient, updateStrategy)
-		testsuite.EnsureKubevirtReadyWithTimeout(kubevirtReadyTimeoutSeconds)
+
+		currentKv := util2.GetCurrentKv(virtClient)
+		tests.WaitForConfigToBePropagatedToComponent(
+			"kubevirt.io=virt-controller",
+			currentKv.ResourceVersion,
+			tests.ExpectResourceVersionToBeLessEqualThanConfigVersion,
+			60*time.Second)
```

Comment:
```suggestion
			time.Minute)
```
Will 1 minute be enough here?

Reply:
@jean-edouard Yeah its pretty much time, the client cache on virt-controller side should receive the updated object by then, if IMO then we have some issue.

Reply:
/approve